pkg/v2/types/types.go

```diff
@@ -118,13 +118,13 @@ func (t *ProvenanceType) UnmarshalJSON(data []byte) error {
 	}
 
 	switch tmp {
-	case "verified_provenance":
-		*t = ProvenanceTypeVerifiedProvenance
+	case "verified_provenance_match":
+		*t = ProvenanceTypeVerified
```

Comment:
... reuse their value here?

Reply:
Do you mean like this?

```suggestion
	case string(ProvenanceTypeVerified):
		*t = ProvenanceTypeVerified
```

Reply:
Cool yes thanks!